src/extension/background.js

```diff
@@ -255,6 +255,30 @@ async function updateHelpContent() {
   }
 }
 
+/**
+ * Checks if the view source popp needs to be openeded, and opens it if necessary.
+ * @param {*} id The tab ID
+ */
+function checkViewSource(id) {
+  chrome.tabs.get(id, (tab = {}) => {
+    if (!tab.url) return;
+    try {
+      const u = new URL(tab.url);
+      const vds = u.searchParams.get('view-doc-source');
+      if (vds && vds === 'true') {
+        chrome.windows.create({
+          url: chrome.runtime.getURL(`/view-source/index.html?tabId=${id}`),
+          type: 'popup',
+          width: 740,
+          height: 1200,
```

Comment:
Would not specifying a `height` default to the parent window's height? Might be safer...

Reply:
Good point. I have removed the height and the browser uses the height of the current browser window.